imports/plugins/included/shipping-rates/server/no-meteor/util/getShippingRestrictionAttributes.js

```diff
@@ -0,0 +1,64 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+import { findCatalogProductsAndVariants, pick, tagsByIds, mergeProductAndVariants } from "./helpers";
+
+/**
+ * @name getShippingRestrictionAttributes
+ * @summary Get shipping attributes for a fulfillment group that will be used to
+ * determine any applicable shipping restrictions.
+ * @param {Object} context -  an object containing the per-request state
+ * @param {Object} totals - The totals object with discounts, item, and group totals
+ * @param {Object} commonOrder - details about the purchase a user wants to make.
+ * @param {Array} fulfillmentGroup.items - the items in the cart
+ * @returns {Object|null} shipping restriction attributes for the provided fulfillment group
+ */
+export default async function getShippingRestrictionAttributes(context, totals, commonOrder) {
+  const { collections, getFunctionsOfType } = context;
+  const { shippingAddress: destination, items: orderItems } = commonOrder;
+  const address = pick(destination, ["address1", "address2", "city", "country", "postal", "region"]);
```

Comment:
Is there a reason we're picking certain props from address? I'd just keep the whole `shippingAddress` as-is. Then you could also delete the `pick` function. (Should by using `_.pick` anyway)

Reply:
@willopez please see @aldeed's question above. I do remember discussing at some point getting rid of that `pick`, but just want to make sure there isn't a reason we kept it in.

Reply:
@kieckhafer that was my oversight, I was not aware that we had _.pick from lodash,  and I agree with @aldeed that the whole object should be be passed and the picking is not necessary. The custom pick function should be removed also.